gateway/api_test.go

```diff
@@ -2054,23 +2110,24 @@ func TestOAS(t *testing.T) {
 	})
 }
 
-func testUpdateAPI(t *testing.T, g *Test, api interface{}, apiID string, oasTyped bool) {
-	updatePath := "/tyk/apis/" + apiID
+func testUpdateAPI(t *testing.T, ts *Test, api interface{}, apiID string, oasTyped bool) {
+	updatePath := "/tyk/apis/"
 	if oasTyped {
-		updatePath += "?type=oas"
+		updatePath += "oas/"
 	}
+	updatePath += apiID
 
-	_, _ = g.Run(t, []test.TestCase{
+	_, _ = ts.Run(t, []test.TestCase{
 		{AdminAuth: true, Method: http.MethodPut, Path: updatePath, Data: &api,
 			BodyMatch: `"action":"modified"`, Code: http.StatusOK},
 	}...)
 
-	g.Gw.DoReload()
+	ts.Gw.DoReload()
 }
 
 func testGetOASAPI(t *testing.T, d *Test, id, name, title string) (oasDoc openapi3.T) {
 
-	getPathWithOASParam := "/tyk/apis/" + id + "?type=oas"
+	getPathWithOASParam := "/tyk/apis/oas/" + id
```

Comment:
`getPathWithOASParam` could be renamed to `getPath` since it's not having a param anymore